README.md

```diff
@@ -1,29 +1,28 @@
 
-Tracking and mapping RPI's shuttles with [Go](https://golang.org/), [Vue.js](https://vuejs.org/), and [MongoDB](https://www.mongodb.org/).
+Tracking and mapping RPI's shuttles with [Go](https://golang.org/), [Vue.js](https://vuejs.org/), and [Postgres](https://www.postgresql.org).
 
 Check it out in action at [shuttles.rpi.edu](https://shuttles.rpi.edu).
 
 ## Setting Up
 
-1. Install Go (https://golang.org/doc/install)
-2. Ensure your `$GOPATH` is set correctly, and is apart of your `$PATH`
+1. [Install Go](https://golang.org/doc/install). Shuttle Tracker targets Go 1.7 and newer, but we recommend using the latest Go stable release.
+2. Ensure your `$GOPATH` is set correctly, and `$GOPATH/bin` is in your `$PATH`
 3. Run `go get github.com/wtg/shuttletracker`
-4. Install `govendor`  by running `go get -u github.com/kardianos/govendor`
+4. Install `govendor` by running `go get -u github.com/kardianos/govendor`
 5. Switch to the Shuttle Tracker directory (`$GOPATH/src/github.com/wtg/shuttletracker`)
 6. Run `govendor sync`
 7. Ensure you have MongoDB installed.
 8. Rename `conf.json.sample` to `conf.json`
 9. Edit conf.json with the following:
    * `DataFeed`: API with tracking information from iTrak... For RPI, this is a unique API URL that we can get data from. It's currently private, and we will only share it with authorized members for now.
    * `UpdateInterval`: Number of seconds between each request to the data feed
-   * `MongoUrl`: URL where MongoDB is located
-   * `MongoPort`: Port where MongoDB is bound (default is 27017)
-10. Start MongoDB, and ensure it is running, and listening on port 27017 (or whichever port you defined in `MongoPort` within `conf.json`)
+   * `Postgres.URL`: URL where Postgres is located
+10. Start Postgres.
 11. Add data to your database. Example DBs are provided in `example_database`, as well as a simple import/export script to setup the database for you.
-    - If using an example database, you might need to check the name of the imported database, and change `MongoUrl` accordingly.
-12. Start the app by running `go run main.go` in the project root directory.
-13. You can optionally add yourself as an administrator by using the `make-admin` script in the example_database folder, passing it your RCS ID as the first argument.
+    - If using an example database, you might need to check the name of the imported database and change `PostgresUrl` accordingly.
+12. Start the app by running `go run cmd/shuttletracker/main.go` in the project root directory.
+13. You can optionally add yourself as an administrator by using the `make-admin.sh` script in the example_database folder, passing it your RCS ID as the first argument.
 14. Visit http://localhost:8080/ to view the tracking application and http://localhost:8080/admin to view the administration panel
 
 ## Configuration
```

Comment:
Are you thinking that I should put a small SQL dump in the repo that can be used to populate a database? I know we also talked about writing some code so that a new Shuttle Tracker instance could just pull down the current routes and stops from shuttles.rpi.edu.

Reply:
I was thinking writing code to pull down the current routes and stops, but if the first option is easier we could also do that for now.